src/LoginScreen.js

```diff
@@ -1,9 +1,61 @@
-import React from 'react';
+import React, {Component} from 'react';
+import {connect} from 'react-redux';
+import {loginWithUserData} from './helperFunctions/login';
+import {updateJWT} from './actions';
+import './stylesheets/login.css';
 
-let LoginScreen = () => 
-    <div>
-        <p>Login Screen</p>
-    </div>
 
 
+let mapStateToProps = (state) => {
+    return {jwt: state.jwt}
+};  
+
+let mapDispatchToProps = (dispatch) => {
+  return {dispatch: dispatch}
+};
+
+class LoginScreenDumb extends Component {
+
+    
+    render() {
+
+        let submitUserLoginInformation = (event) => {
+            event.preventDefault();
+            let username = event.target.username.value;
+            let password = event.target.password.value;
+            event.target.reset();
+            let JSONLoginInfo = {"username":username, "password":password}
+            loginWithUserData(JSON.stringify(JSONLoginInfo))
+            .then(res => this.props.dispatch(updateJWT(res)))
+            .then(this.props.history.push(`/`))
+        };
+
+        let toggleForm = (event) => {
+        };
+
+        let submitNewUserInformation = (event) => {
+            event.preventDefault();
+        };
+
+        return <div className="login-page">
```

Comment:
To clarify, this is what should be set to LoginScreenDumb = ({jwt, dispatch}) => .

The "connect" function will handle all the issues with state.